lib/charms/opensearch/v0/opensearch_backups.py

```diff
@@ -536,6 +536,9 @@ def _generate_backup_list_output(self, backups: Dict[str, Any]) -> str:
 
     def _on_list_backups_action(self, event: ActionEvent) -> None:
         """Returns the list of available backups to the user."""
+        if not self.charm.opensearch_peer_cm.deployment_desc():
+            event.fail("The action can be run only after the deployment is finished.")
```

Comment:
nit: phrasing on all occurrences
```suggestion
            event.fail("The action can only be run once the deployment is complete.")
```